bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/Trajectory/Orbit/Model/Kepler/COE.cpp

```diff
@@ -387,6 +387,37 @@ inline void OpenSpaceToolkitAstrodynamicsPy_Trajectory_Orbit_Model_Kepler_COE(py
             arg("anomaly_type")
         )
 
+        .def_static(
+            "frozen_orbit",
+            &COE::FrozenOrbit,
+            R"doc(
+                Build a `COE` model of a frozen orbit.
```

Comment:
Worth updating the docstrings here to specify what the various combinations of legal inputs are:
- just sma
- sma + inclination
- sma + AoP
- sma + inclination + aop (but some have to be critical angles)
etc.